app/[slug]/(base)/settings/page.tsx

```diff
@@ -1,15 +1,5 @@
-import {TrashIcon} from 'lucide-react'
-import {Button} from '~/components/ui/button'
+import {redirect} from 'next/navigation'
 
-export default function Settings() {
-	return (
-		<div className='flex h-full w-full flex-col gap-4'>
-			<h3>Settings</h3>
-			<Button
-				className='w-fit'
-				variant='destructive'>
-				Delete team <TrashIcon className='h-4 w-4' />
-			</Button>
-		</div>
-	)
+export default async function Settings({params}: {params: {slug: string}}) {
+	redirect(`/${params.slug}/settings/members`)
```

Comment:
Should we just delete this page altogether? And adjust any links to it

Reply:
Fair, good idea but what happens if someone visits `https://maige.app/[team]/settings`? I would imagine some folks would try to open this by entering the URL.